src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -108,6 +111,28 @@ export default class EnhancedReactTable extends React.Component<IEnhancedReactTa
         // binding "this" to handler functions
         this.handleFilterInput = this.handleFilterInput.bind(this);
         this.handleVisibilityToggle = this.handleVisibilityToggle.bind(this);
+        this.onChangeItemsPerPage = this.onChangeItemsPerPage.bind(this);
+        this.onPreviousPageClick = this.onPreviousPageClick.bind(this);
+        this.onNextPageClick = this.onNextPageClick.bind(this);
+    }
+
+    onChangeItemsPerPage(itemsPerPage:number) {
```

Comment:
Can we make these methods private, and move them right after (or right before) the other state change handlers for consistency?